exercises/concept/role-playing-game/.docs/hints.md

```diff
@@ -1,10 +1,23 @@
 # Hints
 
-- See the [F# language reference on Options][options] for more usage examples.
+## 1. Introduce yourself
+
+The `introduce` function must "unwrap" the value contained in `player.Name`.
+Consider using the [`Options.defaultValue` function][defaultvalue] to convert `player.Name` to its "wrapped" value when it is `Some` or the required default value when it is `None`.
+
+## 2. Implement the revive mechanic
+
+The `revive` function must return either `Some <player>` or `None`, where `<player>` is a `Player` record possibly with some modified fields.
 
 ## 3. Implement the spell casting mechanic
 
-- The [`max` function][max] can be used to ensure the health value does not go below 0.
+Consider using a match expression to cover both the `Some` and `None` cases, as in the following example:
+
+```
+let exists (x: Option<int>) =
```

Comment:
The [Microsoft doc whence this example originated](https://learn.microsoft.com/en-us/dotnet/fsharp/language-reference/options) uses `int option` instead of `Option<int>`.  Any reason to cover `int option` in this doc/exercise?

Reply:
Absolutely. I think maybe we should even reverse the approach, where we use `int option` everywhere in the exercise but with a remark that it is a shorthand form of `Option<int>`

Reply:
OK.  Since that is beyond the original scope of this PR, should that change go into a separate PR?  Or should we expand the scope of this one?

Reply:
Absolutely!